index.bs

```diff
@@ -232,22 +232,22 @@ contain=] |hintName|:
  </ol>
 </ol>
 
-When asked to <dfn abstract-op>remove client hints from redirect if needed</dfn> with |request| as input, run the following steps:
+When asked to <dfn abstract-op>update client hints from redirect if needed</dfn> with |request| as input, run the following steps:
 
 <ol>
  <li>If |request|'s [=client=] is null, then abort these steps.
- <li>Let |clientHintsSet| be |request|'s <var ignore>client</var>'s [=environment settings object/client hints set=].
+ <li>Let |clientHintsSet| be the set of all client hints supported by the [=user agent=].
```

Comment:
This ignores the Accept-CH opt in. I don't think this is what you're aiming for, right?